pkg/apis/build/v1alpha1/build_types_test.go

```diff
@@ -113,3 +92,46 @@ func TestBuildGroupVersionKind(t *testing.T) {
 		t.Errorf("GetGroupVersionKind mismatch; expected: %v got: %v", expectedKind, b.GetGroupVersionKind().Kind)
 	}
 }
+
+// dataAs interprets the YAML contents of the file at the given path as the
+// given type.
+func dataAs(relpath string, obj interface{}) error {
```

Comment:
Where is this function being used?

Reply:
Oops, nothing uses it anymore. Originally this PR was to move `buildtest.DataAs` into this test package, but then I realized we didn't need it at all, so we can just get rid of all of it! 🗑